src/IO/TinyPlyLoader/TinyPlyFileLoader.cpp

```diff
@@ -74,6 +75,66 @@ struct memory_stream : virtual memory_buffer, public std::istream {
         memory_buffer( first_elem, size_ ), std::istream( static_cast<std::streambuf*>( this ) ) {}
 };
 
+template <typename DataType, typename ContainerType>
+void copyArrayToRadiumcontainer( const uint8_t* buffer, ContainerType& container, size_t count ) {
+    CORE_UNUSED( buffer );
+    CORE_UNUSED( container );
+    CORE_UNUSED( count );
+    LOG( logERROR )
```

Comment:
Should this error be raised at compile time ? I think so (so use a static_assert for instance)